server/src/main/java/io/spine/server/event/ERepository.java

```diff
@@ -83,7 +83,7 @@ public Event apply(@Nullable EEntity input) {
         final Iterator<EEntity> filtered = filter(entities, detailedLookupFilter);
         final List<EEntity> mutableList = newArrayList(filtered);
         sort(mutableList, comparator());
-        final Iterator<Event> result = Iterators.transform(mutableList.iterator(), getEventFunc());
+        final Iterator<Event> result = transform(mutableList.iterator(), getEvent());
```

Comment:
Please extract the variable `eventEntities` or something like that. `mutableList` is not descriptive.